dev/src/microclimate/project/ProjectCapabilities.ts

```diff
@@ -0,0 +1,94 @@
+/*******************************************************************************
+ * Copyright (c) 2018, 2019 IBM Corporation and others.
```

Comment:
Since this is a new file, the copyright year should just be 2019

Reply:
is fixed